.changeset/bug-native-select.md

```diff
@@ -0,0 +1,5 @@
+---
+'react-magma-dom': patch
+---
+
+Removing SelectTriggerButton to fix dropdown caret issues.
```

Comment:
Think of this changelog as what our adopters see when updating versions. They likely don't care about the implementation (they don't know what SelectTriggerButton is), but they care **what** bug is fixed. So I'd recommend writing these messages in terms of what was fixed instead of how. In addition, it's helpful to note what component is affected.

Consider something along the lines of: `fix(component): Description of bug fix`